packages/node/src/ethereum/install-commitment.ts

```diff
@@ -68,12 +68,13 @@ export class InstallCommitment extends MultiSendCommitment {
       to: this.networkContext.StateChannelTransaction,
       value: 0,
       data: iface.functions.executeAppConditionalTransaction.encode([
-        this.networkContext.AppRegistry,
-        this.networkContext.NonceRegistry,
-        uninstallKey,
-        this.rootNonceValue,
-        appIdentityHash,
-        this.terms
+        /* appRegistry */ this.networkContext.AppRegistry,
+        /* nonceRegistry */ this.networkContext.NonceRegistry,
+        /* uninstallKey */ uninstallKey,
+        /* rootNonceExpectedValue */ this.rootNonceValue,
```

Comment:
If these are different, perhaps rename the JS variable to the expected name of the Solidity variable?

Reply:
please open a separate issue for this if needed